system_launch/launch/system.launch.xml

```diff
@@ -1,84 +0,0 @@
-
-    <!-- Error Monitor -->
-    <group>
-      <let name="config_file" value="$(find-pkg-share system_launch)/config/system_error_monitor/system_error_monitor.param.yaml" if="$(eval &quot;'$(var run_mode)'=='online'&quot;)"/>
-      <let
-        name="config_file"
-        value="$(find-pkg-share system_launch)/config/system_error_monitor/system_error_monitor.planning_simulation.param.yaml"
-        if="$(eval &quot;'$(var run_mode)'=='planning_simulation'&quot;)"
-      />
-      <include file="$(find-pkg-share system_error_monitor)/launch/system_error_monitor.launch.xml">
-        <arg name="config_file" value="$(var config_file)" />
-        <arg name="extra_agg_config_file_sensing" value="$(var sensor_launch_pkg)/config/diagnostic_aggregator/sensor_kit.param.yaml" />
-        <arg name="extra_agg_config_file_system" value="$(find-pkg-share system_launch)/config/diagnostic_aggregator/system.param.yaml" />
-        <arg name="extra_agg_config_file_vehicle" value="$(find-pkg-share system_launch)/config/diagnostic_aggregator/vehicle.param.yaml" />
-        <arg name="use_emergency_hold" value="false" />
-      </include>
-    </group>
-
-    <!-- Emergency Handler -->
-    <group>
-      <let name="config_file" value="$(find-pkg-share system_launch)/config/emergency_handler/emergency_handler.param.yaml"/>
-      <include file="$(find-pkg-share emergency_handler)/launch/emergency_handler.launch.xml">
-        <arg name="config_file" value="$(var config_file)" />
-      </include>
-    </group>
-
-    <!-- MRM Operator -->
-    <group>
-      <let name="config_file" value="$(find-pkg-share system_launch)/config/mrm_comfortable_stop_operator/mrm_comfortable_stop_operator.param.yaml"/>
-      <include file="$(find-pkg-share mrm_comfortable_stop_operator)/launch/mrm_comfortable_stop_operator.launch.py">
-        <arg name="config_file" value="$(var config_file)" />
-      </include>
-    </group>
-    <group>
-      <let name="config_file" value="$(find-pkg-share system_launch)/config/mrm_emergency_stop_operator/mrm_emergency_stop_operator.param.yaml"/>
-      <include file="$(find-pkg-share mrm_emergency_stop_operator)/launch/mrm_emergency_stop_operator.launch.py">
-        <arg name="config_file" value="$(var config_file)" />
-      </include>
-    </group>
-
-    <!-- Dummy Diag Publisher -->
-    <group if="$(eval &quot;'$(var run_mode)'=='planning_simulation'&quot;)">
-      <push-ros-namespace namespace="dummy_diag_publisher"/>
-      <include file="$(find-pkg-share system_launch)/launch/dummy_diag_publisher/control.launch.xml"/>
-      <include file="$(find-pkg-share system_launch)/launch/dummy_diag_publisher/localization.launch.xml"/>
-      <include file="$(find-pkg-share system_launch)/launch/dummy_diag_publisher/system.launch.xml"/>
-      <include file="$(find-pkg-share system_launch)/launch/dummy_diag_publisher/vehicle.launch.xml"/>
-      <include file="$(var sensor_launch_pkg)/launch/dummy_diag_publisher/sensing.launch.xml"/>
```

Comment:
Where is this set of dummy diags for sensors will be launched in the modified version? (Apparently it is missing?)

Reply:
I moved the top three parameters to one parameter file, [autoware_launch/config/system/dummy_diag_publisher/dummy_diag_publisher.param.yaml](https://github.com/tier4/autoware_launch/pull/762/files#diff-80fcd5260bc3f3b8ba9281f0ac3121342d40583115474c15557d85ecc3af1731).


The last params (for sensors) are originally in [tier4/aip_launcher](https://github.com/tier4/aip_launcher/blob/tier4/universe/aip_xx1_launch/launch/dummy_diag_publisher/sensing.launch.xml). I moved these params into [tier4/aip_laucher:<sensor_name>/config/dummy_diag_publisher/sensor_kit.param.yaml](https://github.com/tier4/aip_launcher/pull/101/files#diff-90a7719958a3cd564dca01a51f93327c1ef0fb73f4facc5bacafb09be1093bb5).

Reply:
Sorry, my question didn't make sense.
What I wanted to ask: Is this set of dummy diags for sensors also launched in the modified version? (Apparently it is missing?)

And seems I found the answer by myself, it was launched in teir4_system_launch. So never mind!